source/WebApp/scripts.ts

```diff
@@ -11,6 +11,8 @@ import csso from 'postcss-csso';
 import sharp from 'sharp';
 import htmlMinifier from 'html-minifier';
 import AdmZip from 'adm-zip';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const regexCombiner = require('regex-combiner');

```

Comment:
This can be import as well -- are there any compiler complaints if you do `import regexCombiner from 'regex-combiner';`?

Reply:
Yup, the error was:

> TSError: ⨯ Unable to compile TypeScript:
scripts.ts:15:27 - error TS7016: Could not find a declaration file for module 'regex-combiner'. '/Users/am11/projects/SharpLab/source/WebApp/node_modules/regex-combiner/regex-combiner.js' implicitly has an 'any' type.
  Try `npm install @types/regex-combiner` if it exists or add a new declaration (.d.ts) file containing `declare module 'regex-combiner';`
  
 Other option is to open a PR in DefinitelyTyped repo. I haven't looked into their 'how to PR' mechanism.

Reply:
Thanks -- for now I think we can just do the same as what I did with `postcss-csso`:
https://github.com/ashmind/SharpLab/blob/457058c7bb98c34710863c53d580f94d12649e60/source/WebApp/scripts.ts#L8-L10

Reply:
Much better! 👏